web/api/v1/api.go

```diff
@@ -1479,6 +1443,18 @@ func (api *API) respondError(w http.ResponseWriter, apiErr *apiError, data inter
 	}
 }
 
+func parseTimeParam(r *http.Request, paramName string, defaultValue time.Time) (int64, time.Time, error) {
+	val := r.FormValue(paramName)
+	if val != "" {
```

Comment:
this can be better inlined by inverting the condition.

Reply:
Yes, thanks!